chacra/tests/async/test_rpm.py

```diff
@@ -0,0 +1,24 @@
+import mock
+import pytest
```

Comment:
the flake8 is saying this line is extra:
```
chacra/tests/async/test_rpm.py:2:1: F401 'pytest' imported but unused
```

Reply:
And I wonder why? %-)

Reply:
Fixed